openedx2zim/scraper.py

```diff
@@ -496,7 +507,7 @@ def get_favicon(self):
         favicon_fpath = self.build_dir.joinpath("favicon.png")
 
         # download the favicon
-        save_large_file(self.favicon_url, favicon_fpath)
+        save_large_file(self.favicon_url,favicon_fpath)
```

Comment:
Why did you removed this space?

Reply:
was an accident , made this change as well